### ISSUE TITLE
"parseFromDeveloperNoteTrimCharacters" parameter missing in Sync-XliffTranslations / Test-BcAppXliffTranslations

### ISSUE THREAD

**Issue by fvet** [rvanbekkum/ps-xliff-sync#39]
Hi @rvanbekkum ,

We've finally moved the translation logic to our pipelines and are 99% close to auto-generate translations as part of the build process. However, we lack support for the `parseFromDeveloperNoteTrimCharacters` setting in PS Xliff Sync (although it is supported in VS Code > https://github.com/rvanbekkum/vsc-xliff-sync/issues/100)

Our PS script currently looks like this ...

```
...
$SyncAdditionalParameters = @{
    "preserveTargetAttributesOrder"   = $true
    "findBySource"                    = $true 
    "findBySourceAndDeveloperNote"    = $true
    "parseFromDeveloperNote"          = $true
    "parseFromDeveloperNoteOverwrite" = $true
    "parseFromDeveloperNoteSeparator" = "||"
    # parseFromDeveloperNoteTrimCharacters" = "\""    ### <<< This is currently missing
}

...

Test-BcAppXliffTranslations -printProblems -appFolders $AppFolders -translationRules @("OptionMemberCount", "OptionLeadingSpaces", "Placeholders", "ConsecutiveSpacesConsistent", "ConsecutiveSpacesExist") -restrictErrorsToLanguages @("nl-BE", "nl-NL") -buildProjectFolder $BuildProjectFolder -syncAdditionalParameters $SyncAdditionalParameters -AzureDevOps $AzureDevOps # -reportProgress -FormatTranslationUnit 
```

... but we would like to specify `parseFromDeveloperNoteTrimCharacters` similar to how we can specify this in VS Code.

Any chance of integrating below commit into the PS version of XLiff Sync?
https://github.com/rvanbekkum/vsc-xliff-sync/commit/fc0ba6a4fba177da831bf81101d76e33982286a8

**Comment by rvanbekkum** [comment]
Hi @fvet,

I think that should be possible with limited effort, so I'll add it soon.

**Comment by fvet** [comment]
@rvanbekkum Sorry for being pushy, but any update on the progress of this issue?

**Comment by rvanbekkum** [comment]
Hi @fvet,
I have just pushed a commit which will add support for the `parseFromDeveloperNoteTrimCharacters` parameter.
I will need to test and release it. If you have any feedback then please let me know as well.
The implementation is taken from the vsc-xliff-sync implementation and converted to PowerShell.

**Comment by rvanbekkum** [comment]
The new parameter is now available in version 1.8.0.0.
